backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -162,16 +165,15 @@ def normalize_and_get_doi(body: dict, errors: list) -> Optional[str]:
     return doi
 
 
-def get_publisher_metadata(provider, doi):
+def get_publisher_metadata(doi: str, errors: list) -> Optional[dict]:
     """
     Retrieves publisher metadata from Crossref.
     """
-    provider = crossref_provider.CrossrefProvider()
+    provider = CrossrefProvider()
     try:
         return provider.fetch_metadata(doi)
     except CrossrefDOINotFoundException:
-        # TODO: add an error message
-        raise InvalidParametersHTTPException(detail="DOI cannot be found on Crossref")
+        errors.append({"name": "doi", "reason": "DOI cannot be found on Crossref"})
```

Comment:
You're passing the string "doi" to "name", did you mean to pass the string variable without the quotes?

Reply:
the error message has change.